tests/bluetooth/tester/src/gap.c

```diff
@@ -738,6 +738,47 @@ static void auth_cancel(struct bt_conn *conn)
 	/* TODO */
 }
 
+struct bond_info_args {
+	const bt_addr_le_t *addr;
+	bool bond_found;
+};
+
+static void bond_info(const struct bt_bond_info *info, void *user_data)
+{
+	struct bond_info_args *args = user_data;
+
+	if (bt_addr_le_cmp(&info->addr, args->addr) == 0) {
+		args->bond_found = true;
+	}
+}
```

Comment:
We could consider moving the `bt_addr_le_is_bonded` function to the public API instead of reimplementing the same function here.

Reply:
I see that historically the tester hasn't used the internal header files. Is this by design, or just haven't had a need for it yet? Because alternatively is to just use the internally defined `bt_addr_le_is_bonded` function.

Reply:
I think we just haven't had the need for it. Although I haven't done a lot of work on gap/gatt/l2cap/sm in the tester in Zephyr.